docs/installation_linux.rst

```diff
@@ -46,6 +46,12 @@ Run the command below to install the dependencies on Ubuntu:
 
    sudo apt install git gcc libgirepository1.0-dev libcairo2-dev qemu-utils libvirt-dev python3-pip python3-venv -y
 
+On Azure Linux, run the following command:
```

Comment:
This line is good, but specify above line the same format:
"Run the command below to install the dependencies on Ubuntu:"
to
"On Ubuntu, run the following command:"